### ISSUE TITLE
Prepare swag artwork for Printful

### ISSUE THREAD

**Issue by markoprljic** [GeneralMagicio/gm-planning#60]

**Comment by markoprljic** [comment]
This is what @kristoferlund shared as options we can choose for products, because they have good pricing and availability.

Baseball hat: https://www.printful.com/custom/embroidered/dad-hats/classic-dad-cap-yupoong-6245cm

Tank top: https://www.printful.com/custom/womens/tank-tops/mens-staple-tank-top-bella-canvas-3480

T-shirt: https://www.printful.com/custom/mens/t-shirts/unisex-staple-t-shirt-bella-canvas-3001

Five panel hat: https://www.printful.com/custom/embroidered/hats/5-panel-cap-yupoong-7005

**Comment by markoprljic** [comment]
@jesicadg Updated the issue with Socks! :)

**Comment by markoprljic** [comment]
I recommend that you do the following:

1. Prepare the artwork you need
2. Upload to GM Swag store assets library
3. Add a product from the list above (tshirt, hat etc.)
4. Place artwork on the product

You'll see if you need to adjust it and reupload or not. When you add artwork to assets library in Printful it stays there and you can then place it on any other product, however you may find out that for some products the artwork need to be adjusted.

**Comment by markoprljic** [comment]
Inside the File library in Printful I created a folder for GM, you will find it here. Click on the folder to upload assets there, so they don't get mixed up with all other assets from other projects. 

![Screenshot 2022-05-29 at 12.34.42.png](https://images.zenhubusercontent.com/5cc090818b624e26c3734d75/a6a3664c-512e-4ff1-a294-c73b6477cd57)

**Comment by jesicadg** [comment]
Hi everyone! I left some new product templates in the store. 

**Hats**: attention on final arts here. The Embrodery on the hats have a 6 colour limit.  Had to edit the campfire illustration a bit, let me know what you think. _Is it an option to use the font Akira for "General Magic" embrodery on the side?_
I chose not to use white hats so we can have the logo on white thread.

**Socks**: included one option only for now.

**T-shirts**: worked on a Unisex and an All printed version. I think we're going to have to publish an product for dark fabrics and another for white shirts to change the logo on the back. 

**Stickers**: suggested ones completed, only one missing is ETHCC poster, is that the Paris one? I wasn't sure.

Thanks! Will be working on this while I wait for your comments.

**Comment by markoprljic** [comment]
Hey @jesicadg which store? I can't seem to find it in GM Swag store on Printful.

**Comment by markoprljic** [comment]
> **Hats**: attention on final arts here. The Embrodery on the hats have a 6 colour limit. Had to edit the campfire illustration a bit, let me know what you think. 

Yes I know, I had to do the same :)

>_Is it an option to use the font Akira for "General Magic" embrodery on the side?_ 
Yes we can use both Akira and our Logo, I prefer to use the Logo for the logo, if you know what I mean, and Akira font when we want to write something.

> **Socks**: included one option only for now.
> 
> **T-shirts**: worked on a Unisex and an All printed version. I think we're going to have to publish an product for dark fabrics and another for white shirts to change the logo on the back.
I understand.
 
> **Stickers**: suggested ones completed, only one missing is ETHCC poster, is that the Paris one? I wasn't sure.

Yes it's for Paris this July.

**Comment by jesicadg** [comment]
> Hey @jesicadg which store? I can't seem to find it in GM Swag store on Printful.

Right @markoprljic that's on Product templates in GM Swag Shop

<img width="1436" alt="Screen Shot 2022-05-31 at 07 42 06" src="https://user-images.githubusercontent.com/105446617/171155998-6c3e6788-78ae-41a6-81aa-e34228b9855c.png">

**Comment by markoprljic** [comment]
Ahaaa ok got it.

**Comment by markoprljic** [comment]
@jesicadg Any chance you would be able to finish adding products to swag shop this week? We would need to add product details and make an order for ETH Barcelona.

I also mentioned you in the swag shop discord channel regarding a hat. Check it out! :)

**Comment by jesicadg** [comment]
> @jesicadg Any chance you would be able to finish adding products to swag shop this week? We would need to add product details and make an order for ETH Barcelona.
> 
> I also mentioned you in the swag shop discord channel regarding a hat. Check it out! :)

Yes! The hat is super cool, I think I might need an extra day but I'm on it.  New templates will be up tomorrow afternoon.

**Comment by markoprljic** [comment]
Perfect, thanks!

**Comment by markoprljic** [comment]
![Screenshot 2022-06-10 at 09.19.50.png](https://images.zenhubusercontent.com/5cc090818b624e26c3734d75/1bf67ed0-72f4-46d4-8599-5d6ba2135117)
@jesicadg I'd put this on the back and on the front upper left side just our logo.

**Comment by markoprljic** [comment]
It would be cool to have this creature as a sticker with our logo, similar to campfire girl.

**Comment by rbsal97** [comment]
Hi @markoprljic,

For new designs, what I think is that we create new designs based on an upcoming event and release it in bulk before the event so that any of us can order from the swag shop. We can then decide 
1. What event to follow to reference our designs
2. Deadlines for the bulk designs

Or we can just do what is initially planned and create designs and upload them once finished?

@jesicadg

**Comment by markoprljic** [comment]
@rbsal97 Yeah, this is what I did for Barcelona and Paris (did not attend unfortunately). In the future we can easily create an event specific swag. 

We should now continue creating products based on the artwork we have and just start selling it to general public.

**Comment by rbsal97** [comment]
@tomislavmamic
